### PR TITLE
Remove reference to Gen3 on anvilproject.org/learn home

### PR DESCRIPTION
**Pull request by avahoffman**
Hi all,

We got some comments that this was a bit out of date, and we need to remove reference to Gen3. Attempting to keep things up to date here.

CC @nakucher @cutsort

### CODE REVIEW

File: content/learn/learn.md [hunk 1]
```diff
@@ -12,8 +12,7 @@ an [NHGRI](https://www.genome.gov/Funded-Programs-Projects/Computational-Genomic
 and controlled access genomic [datasets](https://anvilproject.org/data/consortia) with popular analysis tools in
 a [secure](/overview/security) cloud computing environment.
 
-AnVIL uses [Terra](https://anvil.terra.bio/#workspaces) as its analysis platform, [Gen3](https://gen3.theanvil.io) for
```
Comment (NoopDog) [inline content/learn/learn.md:15]: Thanks, @avahoffman, for this PR! Can you reference https://explore.anvilproject.org/ as the data search and artificial cohort creation platform? Cheers, D

File: content/site-map.yaml [hunk 2]
```diff
@@ -301,8 +301,6 @@
               file: "/learn/guides-and-tutorials/guides-and-tutorials"
             - name: "Introduction to Terra"
               file: "/learn/intro-to-terra/intro-to-terra"
-            - name: "Introduction to Gen3"
-              file: "/learn/accessing-data/getting-started-with-gen3"
```
Comment (NoopDog) [inline content/site-map.yaml:305]: Thanks! Can you delete this file as well please? 
Cheers,
D

**Review by NoopDog — CHANGES_REQUESTED** [review]
Thanks so much for this PR @avahoffman. I added a small nits in the review.
Cheers,
D

File: content/learn/learn.md (hunk 1 above)
Comment (avahoffman) [reply content/learn/learn.md:15]: you got it. Change incoming!

File: content/site-map.yaml (hunk 2 above)
Comment (avahoffman) [reply content/site-map.yaml:305]: Is there any concern about directing to a broken link from an external page? Happy to remove if not.

Comment (NoopDog) [reply content/site-map.yaml:305]: Ok we can clean that up. Thx.

**Review by NoopDog — APPROVED** [review]

File: content/site-map.yaml (hunk 2 above)
Comment (avahoffman) [reply content/site-map.yaml:305]: Ok, removed refs to that page!

**Comment by avahoffman** [comment]
Thanks for the quick review @NoopDog !

**Pull request merged**